Add Ceres products to SORA docs (#225)
sora-xor/sora-docs: src/ceres/hermes-dao.md

---
title: "Hermes DAO | SORA Docs"
head:
  - - meta
    - name: description
      content: "Hermes DAO is a decentralized fund for investments in cryptocurrency assets."
  - - meta
    - name: keywords
      content: "Ceres, Hermes, DeFi, decentralized fund, DAO, governance, Polkaswap, SORA network"
---

> Hermes (HMX) is the native token of [Hermes DAO](https://hermes-dao.io/), a decentralized fund for investments in cryptocurrency assets (tokens, coins, NFTs...)

Reviewer: ```suggestion
Hermes (HMX) is the native token of the [Hermes DAO](https://hermes-dao.io/), a decentralized investment fund for cryptocurrency assets (tokens, coins, NFT, etc.)
```

managed by the Hermes DAO Parliament and the community. 
The fund is primarily used for investments in existing crypto projects and projects in early stages.
It is an ideal balance between a hedge fund and high risk investment venture capital with constant and stable returns. 

# Hermes DAO Governance

> [Hermes DAO Governance](https://hermes-dao.io/governance) platform is a tool for implementing the concept of decentralization in the Hermes DAO. 

Reviewer: ```suggestion
The [Hermes DAO Governance](https://hermes-dao.io/governance) platform is a tool to implement the concept of decentralization in the Hermes DAO. 
```

> Any Hermes (HMX) token holder with more than a set limit will be able to create a poll. Likewise, all voters holding Hermes (HMX) tokens above a set minimum have the same voting power. Each wallet's vote counts as one vote. This form of governance brings a greater degree of democracy.

Reviewer: ```suggestion
Any Hermes (HMX) token holder above the set limit of tokens can create a poll. Likewise, all voters holding more Hermes (HMX) tokens than the specified minimum have the same voting power. Each wallet's vote counts as one vote. This form of governance brings a greater degree of democracy.
```

Reviewer: What is the limit? How is it defined?

---
title: "Hermes DAO | SORA Docs"
head:
  - - meta
    - name: description
      content: "Hermes DAO is a decentralized fund for investments in cryptocurrency assets."
  - - meta
    - name: keywords
      content: "Ceres, Hermes, DeFi, decentralized fund, DAO, governance, Polkaswap, SORA network"
---

> 

Reviewer: ```suggestion
# Hermes DAO 
```
